tools/generator/src/Generator+AddTargets.swift

```diff
@@ -644,11 +632,13 @@ chmod u+w "$header"
         return #"""
 \#(copyGeneratedHeader)\#
 # Copy swiftmodule
+SAVEIFS=$IFS; IFS=$'\n'
+swiftmodule=($BAZEL_OUTPUTS_SWIFTMODULE)
+IFS=$SAVEIFS
```

Comment:
@jpsim do you know if there is there a way to do this another way? Arrays in build settings seem to just be whitespace separated, so if I use those I don't think I can support paths with spaces.

Reply:
If I do it this way, I guess this is probably better:
```suggestion
IFS=$'\n' read -r -a swiftmodule <<< "$BAZEL_OUTPUTS_SWIFTMODULE"
```

Reply:
Your revised version looks right to me.

Reply:
Agreed

Reply:
I couldn't get it to read more than one element this way. I'm just leaving it as is for now.